server/router.js

```diff
@@ -45,6 +45,7 @@ router.on('/:archiveKey', {
     }, () => {
       cancelled = true
       console.log('server getArchive() timed out for key: ' + params.archiveKey)
+      state.archive.entries = []
```

Comment:
whoops -- i left it because it doesn't hurt, you can remove directly from master if you feel strongly bc i just merged this.